internal/apiconfig/memory_provider.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
```

Comment:
is it totally necessary though? With our suite of DB backed tests?

Reply:
Probably not? I also think it would be helpful for anyone looking at the code to see two implementations of the interface when asking "why is this an interface?".